data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+			`	"Are you Captain <last>?" one of them asks you, "Don't worry, you're not in trouble or anything."`
+			`	You nod your affirmation.`
+			`	"Your reputation precedes you Captain. As a person of note at home who we hear has now been travelling around Hai space, you are presently being treated as an Envoy."`
+			`	For a second you wonder if it's a joke but they look as serious as Hai can look.`
+			choice
+				`	"Oh I really don't think that's necessary."`
+					goto nevertheless
+				`	"Oh, I'm honoured, thank you."`
+			`	"If you have any stake in the current information crisis please head directly to Hai-home. Someone would very much like to talk to you."`
+				goto main
+			label nevertheless
+			`	"You may not think it's necessary, but sometimes that is the price of fame."`
+			`	The officious-looking Hai shrugs, "In any event please head directly to Hai-home. Someone would very much like to talk to you."`
+			label main
+			`	They leave without waiting for your agreement, but then, you did come here to see what was going on.`
+				accept
+
+
+
+#All prior branches merge here.
+	
+mission "Hai Leaks Response 1"
+	landing
+	name "Hai message to <planet>"
+	description "Take the diplomatic ID from the box from Sayari to <destination>."
+	source "Hai-home"
+	destination "Earth"
+	to offer
+		or
+			has "Hai Tour [5] Tour Ends: done"
+			has "ZZ Event Hai Secret Leaks: RSHU: done"
+			has "ZZ Event Hai Secret Leaks: FWHU: done"
+	on offer
+		log "People" "Sayari" `Ambassador Sayari, a Hai, served as their envoy to the human government a century ago (although no public records of this exist in human space). Her experience has placed her in charge of coordinating the Hai response to the crisis of their continued secrecy.`
+		log `The Hai are very real, and the worlds of humanity seem poised to become aware of this but the Hai would like to avoid this. As a person of note with respect in all corners of human space the Hai have asked you to assist in representing their interests where travel to human space is required as a somewhat neutral party.`
+		conversation
+			`As you come into land on Hai-home you find that a reasonable security detachment has been assigned to your designated landing berth. The Hai aren't usually ones to be concerned about such things, but you suspect that as an apparent "war hero" that this may be more for the benefit of appearances on your part than anything else.`
+			`	Word of your status in human space seems to have spread ahead of you, especially with you being known on many Hai worlds as one of the few significant human visitors. A small crowd of Hai have gathered to peek at you on the short walk from the Spaceport to the government building you are led to.`
+			branch karma
+				karma > 1
+			`	As you are making the walk you hear questions posed and rumours stated. Occasionally you hear words like "survivor" and "dangerous" thrown in amongst the babble.`
```

Comment:
I am inclined to instead add some scattered humans into the crowd that might be the cause for questions to be asked in the standard human tongue.

Reply:
We're on Hai-home, right? Isn't there little to no human presence here?

Reply:
Correct, but given how well travelled Hai space is I assume *little* is the correct term rather than none, and what event is more likely to bring them out to see than the intersection of the politics of their old home and their new?

Reply:
Even if there are _some_ humans on Hai-home, I would imagine the Hai would so far outnumber them that the general babble from everyone would drown out any minority of conversations happening in a human language. That's the last point I'll make though, so you can keep it if you're pretty dead-set on it. I'd imagine this is trivial enough that it's not really worth either of our time continuing to discuss it unless someone else brings it up.

Reply:
What I will say is that, as someone who has spent a good bit of time in foreign countries, one person speaking your language in a crowd of another tongue will stand out amidst the babble really really clearly. Heck, someone speaking a second language I was barely competent at stood out in a crowd enough for me to catch what was said.
It's also possible that the Hai have heard the news through copied human broadcasts, so they could also be bringing up the human word as clarity for the translation or context in a sentence. This would be a not unreasonable reason for you to hear such words even amongst a conversation conducted principally in another language.
In any event, you're correct that it's fairly trivial, and largely for effect.